src/Angor/Avalonia/Angor.Contexts.Funding/FundingContextServices.cs

```diff
@@ -56,8 +59,26 @@ public static ServiceCollection Register(ServiceCollection services, ILogger log
         services.TryAddSingleton<IInvestmentTransactionBuilder, InvestmentTransactionBuilder>();
         services.TryAddSingleton<ITaprootScriptBuilder, TaprootScriptBuilder>();
         services.TryAddSingleton<IWalletOperations, WalletOperations>();
+        services.AddSingleton(GetNostrQueryClient);
         services.AddHttpClient();
 
         return services;
     }
+
+    private static NostrQueryClient GetNostrQueryClient(IServiceProvider provider)
+    {
+        var nc = provider.GetRequiredService<INetworkConfiguration>();
```

Comment:
I think this kind of business logic should be in a service, it's too explicit and related to the internals of a NuGet package we consume for nostr-client

Reply:
Yes, I agree. Anyways, I've removed the registration completely for now because, in the end, the client isn't used at all. We might even remove it completely from this PR and add it in more scoped PR. I'm good with both. What do you prefer?